sonar-scanner-engine/src/main/java/org/sonar/scanner/issue/DefaultIssueCallback.java

```diff
@@ -68,12 +58,10 @@ public void execute() {
       collectInfo(issue);
     }
 
-    getUsers();
-
     for (TrackedIssue issue : issues.all()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Combine this loop with the one that starts on line 57. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3047)